kolibri/plugins/device/assets/src/views/FacilitiesPage/FacilitiesTasksPage.vue

```diff
@@ -39,13 +39,13 @@
 
 <script>
 
+  import { FacilityTaskPanel } from 'kolibri.coreVue.componentSets.sync';
```

Comment:
How do you feel about experimenting with putting some common code (but not really core code) into a module inside `packages/kolibri-common` or some such?

Should then be referenceable as `kolibri-common/path/to/component`.

Reply:
If that would help lighten our default front-end bundle load at all with tree shaking, then I think it's a good call for sure. Otherwise I'd be hesitant to separate anything from the apiSpec since, in spite of it being a big monolith, is an easy thing to reason about and change - at least it is once it's clear the object maps to the aliased paths in imports.

Reply:
Yes - I think a fair amount of the code that is in the default frontend bundle falls into this category, so if we can experiment with this here, it would set us up to migrate more things out of the frontend bundle, and reduce the size considerably, I think.